docs/source/top_level_elements/originInfo_dates.rst

```diff
@@ -0,0 +1,282 @@
+originInfo: dates
+=================
+
+About
+-----
+
+This section describes all of the different subelements of originInfo in our MODS.
+
+Dates
+-----
+
+We have three different date-related subelements: dateCreated, dateIssued, and dateOther.
```

Comment:
Technically we have four, right (with copyrightDate)? I know it's an edge case.

Reply:
Thanks for catching this!